### DISCUSSION TITLE
Purge git lfs files

### DISCUSSION THREAD

**Discussion by lucaro** [vitrivr/cineast#216] [General]
Even though git lfs is not used anymore, there are still files in the history that can cause all kinds of strange issues and continuously cause the bandwidth quota to be exceeded. I don't quite know what can be done to fix this, it might require a history rewrite in order to completely get rid of them. Thoughts?

**Comment by silvanheller** [comment]
I like it, I think we could start collecting wishes for a `clean git history` and then see if there is ever a good time to significantly rewrite the commit tree.
